app/uk/gov/hmrc/lisaapi/ApplicationRegistration.scala

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/*
+
```

Comment:
Commented out file? if it's not needed, it can be deleted!

Reply:
I thought i had deleted it